### PR TITLE
Reduce container image size

### LINKED ISSUE #482: Reduce container image size
Had to switch off of the Node.js Alpine image base because of a dependency issue with Prisma. Would be nice to go back to ~100 MB image sizes instead of 5x that.

### PR DESCRIPTION
**Pull request by bokherus**
Closes #482 

Switch the base image from `node:16.13.0` to `node:lts-alpine`

- [x] I updated the changelog

### CODE REVIEW

**Comment by bokherus** [comment]
I'm not sure if there's any complications to use alpine linux as the base image instead. But the file were much smaller
Screen Shot 2565-01-21 at 21 27 18

**Comment by codetheweb** [comment]
> Could you please explain what's the dependency issue with Prisma? I tried building the image and run it and it seems fine.

Huh, the issue I was having was that Prisma couldn't figure out the correct binary to download (I think OpenSSL might have been missing?). Not sure what changed.

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM node:16.13.0 AS base
+FROM node:lts-alpine AS base
```
Comment (codetheweb) [inline Dockerfile:1]: Please keep the version pinned to v16

**Comment by codetheweb** [comment]
LGTM, thanks.

**Pull request merged**